Cargo.toml

```diff
@@ -885,6 +888,7 @@ mqtt-integration-tests = ["sinks-mqtt"]
 nats-integration-tests = ["sinks-nats", "sources-nats"]
 nginx-integration-tests = ["sources-nginx_metrics"]
 opentelemetry-integration-tests = ["sources-opentelemetry", "dep:prost"]
+openobserve-integration-tests = ["sinks-openobserve"]
```

Comment:
I don't see any use of `openobserve-integration-tests` in this PR. Can we delete it here and from the github workflows?

Reply:
yes